awx/main/scheduler/task_manager.py

```diff
@@ -109,7 +109,12 @@ def record_aggregate_metrics_and_exit(self, *args):
         self.record_aggregate_metrics()
         sys.exit(1)
 
-    def schedule(self):
+    def schedule(self, debug=False):
+
+        if settings.SETTINGS_MODULE == 'awx.settings.development' and settings.AWX_DISABLE_TASK_MANAGERS and not debug:
+            logger.debug(f"Not running {self.prefix} scheduler, AWX_DISABLE_TASK_MANAGERS is True. Trigger with GET to /api/{self.prefix}_manager_debug/")
+            return
```

Comment:
instead of the debug kwarg, it seems like it would be easier to have this exit condition in the `@task` method

Reply:
ok